src/vos/vos_ilog.h

```diff
@@ -133,6 +133,23 @@ vos_ilog_check_(struct vos_ilog_info *info, const daos_epoch_range_t *epr_in,
 void
 vos_ilog_desc_cbs_init(struct ilog_desc_cbs *cbs, daos_handle_t coh);
 
+/** Aggregate (or discard) the incarnation log in the specified range
+ *
+ * \param	coh[IN]		container handle
+ * \param	ilog[IN]	Incarnation log
+ * \param	epr[IN]		Aggregation range
+ * \param	discard[IN]	Discard all entries in range
+ * \param	punched[IN]	Highest epoch where parent is punched
+ * \param	info[IN]	Incarnation log info
+ *
+ * \return	0	Success
+ *		error	Failure
```

Comment:
The ilog_aggregate() (that is main body for vos_ilog_aggregate) can return positive value.